packages/tools/devtools/devtools-test-app/src/App.tsx

```diff
@@ -95,7 +95,7 @@ function useContainerInfo(
 			devtools.registerContainerDevtools({
 				container: containerInfo.container,
 				containerKey: sharedContainerKey,
-				containerData: containerInfo.appData.getRootObject(),
+				containerData: { appData: containerInfo.appData },
```

Comment:
Oh, I was also curious why this one was necessary?

Reply:
Until now, we have passed in the `AppData._initialObjects: Record<string, IFluidLoadable>`. But now that we are able to visualize `DataObject`, we are starting our visualization from the root `DataObject` *not* the root object. 

And you can also see from the picture above in description, that the visualizer now renders type `DataObject` (in red) for `appData` and `shared-data-object` 😄 .